kubernetes/fineractmysql-deployment.yml

```diff
@@ -96,11 +96,7 @@ spec:
                   key: password
           livenessProbe:
             exec:
-              command:
-                - mysqladmin
-                - ping
-                - -h
-                - localhost
+              command: ["sh","-c","mysqladmin ping -h localhost -uroot -p${MYSQL_ROOT_PASSWORD}"]
```

Comment:
NB this change here was already in #1583 (for FINERACT-1294), so this diff should disappear on a rebase.